dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -216,9 +218,9 @@ public void testComponent_staticLinkageCheckResult() throws IOException, Parsing
 
       Nodes staticLinkageCheckMessage = document.query("//p[@class='jar-linkage-report']");
```

Comment:
This variable isn;'t very well named, now that I look at it. Maybe just `reports`

Reply:
Renamed to `reports`